### PR TITLE
[AdherentBinome] Fix style on "Mes activités"

### PR DESCRIPTION
**Pull request by enzozccr**
Need someone to :
- dynamise tabs "Mes comités / Mes événements"
- dynamise sub-tabs  "Passés / A venir" (events)

### CODE REVIEW

File: templates/adherent/committees.html.twig [hunk 1]
```diff
@@ -4,74 +4,83 @@
...
+                        </a>
                         <button role="button"
-                                class="btn btn--blue committee-action-button"
+                                class="btn btn--ghosting--blue committee-action-button"
                                 id="committee-toggle-membership-{{ loop.index0 }}"
                                 data-action="quitter"
                                 data-slug="{{ membership.committee.slug }}"
                                 data-csrf-token="{{ csrf_token('committee.unfollow') }}">
                             <img src="{{ asset('images/loader-sm.gif') }}" class="btn-loader visually-hidden" />
-                            Quitter ce comité
```
Comment (Tiriel) [inline templates/adherent/committees.html.twig]: Seen with Guillaume, the wording was to be changed to "Quitter ce comité", to keep consistency with the "show committee" action iirc.

File: templates/adherent/committees.html.twig [hunk 2]
```diff
@@ -4,74 +4,83 @@
...
+                        {% set votingCommitteeName = membership.committee.name %}
+                            <div class="em-switch__wrapper">
+                                <button role="button"
+                                        class="committee-action-button em-switch__toggle checked"
+                                        id="committee-toggle-vote-{{ loop.index0 }}"
+                                        data-action="ne-plus-voter"
+                                        data-slug="{{ membership.committee.slug }}"
+                                        data-csrf-token="{{ csrf_token('committee.vote') }}">
+                                        <span class="btn-loader"></span>
+                                    <!-- <img src="{{ asset('images/loader-sm.gif') }}" class="btn-loader visually-hidden" /> -->
```
Comment (ottaviano) [inline templates/adherent/committees.html.twig]: maybe remove this?

File: templates/adherent/committees.html.twig [hunk 3]
```diff
@@ -4,74 +4,83 @@
...
-                            Voter dans ce comité
-                        </button>
+                            {% if votingCommitteeName is not empty %}
+                                <div class="em-switch__wrapper">
+                                    <button role="button"
+                                            class="committee-toggle-button em-switch__toggle"
+                                            id="committee-toggle-vote-{{ loop.index0 }}"
+                                            data-slug="{{ membership.committee.slug }}">
+                                            <span class="btn-loader"></span>
+                                        <!-- <img src="{{ asset('images/loader-sm.gif') }}" class="btn-loader visually-hidden" /> -->
```
Comment (ottaviano) [inline templates/adherent/committees.html.twig]: same

File: templates/adherent/committees.html.twig [hunk 4]
```diff
@@ -4,74 +4,83 @@
...
+                            {% else %}
+                                <div class="em-switch__wrapper">
+                                    <button role="button"
+                                            class="committee-action-button em-switch__toggle"
+                                            id="committee-toggle-vote-{{ loop.index0 }}"
+                                            data-action="voter"
+                                            data-slug="{{ membership.committee.slug }}"
+                                            data-csrf-token="{{ csrf_token('committee.vote') }}">
+                                            <span class="btn-loader"></span>
+                                        <!-- <img src="{{ asset('images/loader-sm.gif') }}" class="btn-loader visually-hidden" /> -->
```
Comment (ottaviano) [inline templates/adherent/committees.html.twig]: here too

File: templates/adherent/committees.html.twig [hunk 5]
```diff
@@ -81,19 +90,30 @@
...
-                    data-action="voter"
-                    data-csrf-token="{{ csrf_token('committee.vote') }}">
-                <img src="{{ asset('images/loader-sm.gif') }}" class="btn-loader visually-hidden" />
-            Changer de comité de vote
-            </button>
+            <div class="font-roboto">
+                <div class="text--bold text--default-large">Changement du comité de vote</div>
+                <p class="b__nudge--top-15 b__nudge--bottom-large text--dark">
+                    Vous êtes sur le point de changer votre comité de vote. Vous ne pourrez plus voter dans le comité {{ votingCommitteeName }},
+                    est vous sûr de vouloir changer votre comité de vote ?
```
Comment (ottaviano) [inline templates/adherent/committees.html.twig]: `êtes-vous`

**Pull request merged**